backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -219,7 +219,7 @@ public String getTestResult() {
 
 	@JsonProperty("Specimen_collection_date_time")
 	public String getSpecimenCollectionDateTime() {
-		return dateToHealthCareString(convertToLocalDate(testEvent.getCreatedAt()));
+		return dateToHealthCareString(convertToLocalDate(testEvent.getTestOrder().getDateTested()));
```

Comment:
This is where I would have commented hat it seemed like we should actually have this column on TestEvent rather than TestOrder... 🤪 

We can always move it next week.